test/dummy/config/initializers/mountain_view.rb

```diff
@@ -1,3 +1,6 @@
 MountainView.configure do |config|
   config.included_stylesheets = ["global"]
 end
+
+# include component classes
+Dir.glob('app/components/**/*.rb') { |f| require_relative "../../#{f}" }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.